components/gcp-click-to-deploy/src/DeployForm.tsx

```diff
@@ -121,6 +124,13 @@ export default class DeployForm extends React.Component<any, DeployFormState> {
     // be able to click submit until the fetches have succeeded. How can we do
     // that?
 
+    const values = queryString.parse(this.props.location.search);
```

Comment:
You can instead do something like"
```suggestion
    const params = new URLSearchParams(this.props.location.search);
    if (params.get('version')) {
      ...
```

Reply:
done.